lib/matplotlib/axes/_axes.py

```diff
@@ -384,6 +409,208 @@ def legend(self, *args, **kwargs):
     def _remove_legend(self, legend):
         self.legend_ = None
 
+    def inset_axes(self, rect, transform=None, zorder=5,
```

Comment:
should be kw-only (`(self, rect, transform=None, *, zorder=5, ...)`) because passing zorder positionally would just be hard to read (and, from our PoV, make it harder if we want to change the signature).  same below re: inset_rectangle, etc.

Reply:
ok, I'm always confused about positional arguments.... I don't need `transform` to be on either.  So just putting `*` in there makes all the rest of the args kwarg only?  Thats kind ugly ;-)

Reply:
https://www.python.org/dev/peps/pep-3102/